setup.cfg

```diff
@@ -0,0 +1,135 @@
+[metadata]
+name = django_peertube_runner_connector
+description = A django application to connect to a peertube runner and transcode videos
+long_description = file: README.md, LICENSE
+long_description_content_type = text/markdown
+version = 1
+author = Open FUN (France Universite Numerique)
+author_email = fun.dev@fun-mooc.fr
+license = MIT
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Education
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3 :: Only
+keywords =
+    peertube
+    transcode
+url = https://github.com/openfun/django-peertube-runner-connector
+requires_python = >=3.8
+
+[options]
+install_requires =
+    django==4.2
+    djangorestframework==3.14
```

Comment:
With this requirement it's still possible to install this lib with django 4.1 ? Maybe the selector should be more permissive ? like `Django>=3.2` for example ?